src/main/java/org/commcare/formplayer/db/migration/V18__init_shedlock.java

```diff
@@ -0,0 +1,18 @@
+package org.commcare.formplayer.db.migration;
+
+import java.util.Arrays;
+
+public class V18__init_shedlock extends BaseFormplayerMigration {
+
+    @Override
+    public Iterable<String> getSqlStatements() {
+        return Arrays.asList("CREATE TABLE shedlock (\n" +
+                "    name VARCHAR(64) NOT NULL,\n" +
+                "    lock_until TIMESTAMP NOT NULL,\n" +
```

Comment:
This table schema is an implicit dependency from the new `import net.javacrumbs.shedlock.core.LockProvider` library, right? 

Can we provide some future proofing documentation to outline where the structure came from? It makes me a bit uneasy for someone reviewing this later to not have breadcrumbs to indicate the source of truth for the table schema.